demo/assets/airplane/lode-config.json

```diff
@@ -1,13 +1,7 @@
 {
   "levels": [
     {
-      "threshold": 100
-    },
-    {
-      "threshold": 50,
-      "configuration": {
-        "targetScale": 0.125
-      }
+      "threshold": 300
     },
     {
       "threshold": -1,
```

Comment:
higher target scale does not work nicely?

Reply:
It's more about the change in color / lightning

Reply:
⚡ 
you've added a bunch of new models and they have pretty high polycount - even when optimized. I'm just wondering whether you've tried it as we certainly have to tune it when running the benchmark.